### ISSUE TITLE
coco数据结上表现和官方差两个点

### ISSUE THREAD

**Issue by pengyang1225** [bubbliiiing/yolox-pytorch#36]
对比了官方的s模型，那边map可以道40.5，您这边只有38.2，有对比原因吗

**Comment by bubbliiiing** [comment]
之前batchnor实现有问题，现在纠正了，但是readme没改。这个我回复过…昨天。

**Comment by pengyang1225** [comment]
谢谢，您哪里回复了，您修改了哪里

**Comment by bubbliiiing** [comment]
batch nor的epsilon，pytorch默认是1e-5，应该是1e-3
        self.bn     = nn.BatchNorm2d(out_channels, eps=0.001, momentum=0.03)

**Comment by bubbliiiing** [comment]
B站还是CSDN忘了，消息太多
